<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lumix Studio v0.2</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>892</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionLook_at_selected_entity"/>
    <addaction name="actionSave_Layout"/>
    <addaction name="actionPolygon_Mode"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionGame_mode"/>
    <addaction name="actionCompile_scripts"/>
    <addaction name="actionMeasure"/>
    <addaction name="actionSnap_to_terrain"/>
   </widget>
   <widget class="QMenu" name="menuEntity">
    <property name="title">
     <string>Entity</string>
    </property>
    <addaction name="actionCreate"/>
    <addaction name="actionRemove"/>
    <addaction name="actionSave_as_template"/>
    <addaction name="actionInstantiate_template"/>
    <addaction name="actionHide"/>
    <addaction name="actionShow"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuSelect">
     <property name="title">
      <string>Select</string>
     </property>
     <addaction name="actionSame_mesh"/>
    </widget>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionCenter_Pivot"/>
    <addaction name="actionLocal_Global"/>
    <addaction name="menuSelect"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuEntity"/>
  </widget>
  <action name="actionLog">
   <property name="text">
    <string>Log</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionGame_mode">
   <property name="text">
    <string>Game mode</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionCompile_scripts">
   <property name="text">
    <string>Compile scripts</string>
   </property>
  </action>
  <action name="actionCreate">
   <property name="text">
    <string>Create</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="text">
    <string>Properties</string>
   </property>
  </action>
  <action name="actionGame_view">
   <property name="text">
    <string>Game view</string>
   </property>
  </action>
  <action name="actionScene_View">
   <property name="text">
    <string>Scene View</string>
   </property>
  </action>
  <action name="actionAsset_Browser">
   <property name="text">
    <string>Asset browser</string>
   </property>
  </action>
  <action name="actionFile_server">
   <property name="text">
    <string>File server</string>
   </property>
  </action>
  <action name="actionScript_compiler">
   <property name="text">
    <string>Script compiler</string>
   </property>
  </action>
  <action name="actionMaterial_manager">
   <property name="text">
    <string>Material editor</string>
   </property>
  </action>
  <action name="actionProfiler">
   <property name="text">
    <string>Profiler</string>
   </property>
  </action>
  <action name="actionPolygon_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Wireframe</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionLook_at_selected_entity">
   <property name="text">
    <string>Look at selected entity</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
  </action>
  <action name="actionSnap_to_terrain">
   <property name="text">
    <string>Snap to terrain</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionSave_as_template">
   <property name="text">
    <string>Create template</string>
   </property>
  </action>
  <action name="actionEntity_templates">
   <property name="text">
    <string>Entity templates</string>
   </property>
  </action>
  <action name="actionInstantiate_template">
   <property name="text">
    <string>Instantiate template</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionEntity_list">
   <property name="text">
    <string>Entity list</string>
   </property>
  </action>
  <action name="actionMeasure">
   <property name="text">
    <string>Measure</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionFiles">
   <property name="text">
    <string>Files</string>
   </property>
  </action>
  <action name="actionSave_Layout">
   <property name="text">
    <string>Save Layout</string>
   </property>
  </action>
  <action name="actionCenter_Pivot">
   <property name="text">
    <string>Center/Pivot</string>
   </property>
  </action>
  <action name="actionLocal_Global">
   <property name="text">
    <string>Local/Global</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionSame_mesh">
   <property name="text">
    <string>Same mesh</string>
   </property>
  </action>
  <action name="actionHide">
   <property name="text">
    <string>Hide</string>
   </property>
  </action>
  <action name="actionShow">
   <property name="text">
    <string>Show</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
